src/components/tiles/TilesList.tsx

```diff
@@ -31,9 +33,10 @@ export function TilesList({ navigation, styles: instanceStyles }: Props) {
 
   const onCircleTilePress = useCallback(
     (circleTile: CircleTile) => () => {
-      navigation.navigate('Home/Circle/Discussion', { circleTile });
+      setCircleTile?.(circleTile);
+      navigation.navigate('Home/Circle/Discussion');
```

Comment:
I feel like we should move this inside the circle screen. I think it would be better to set the active circle based on the route params. That way other sources, such as notification, can just pass the circleId when navigating to the circles view.

Reply:
Updated